README.md

```diff
@@ -1,6 +1,31 @@
 [![Project stage][Stage]][Stage-Page]
 [![Release Artifacts][Badge-SonatypeReleases]][Link-SonatypeReleases] [![Sonatype Nexus (Snapshots)](https://img.shields.io/nexus/s/dev.zio/zio-kafka_2.13?server=https%3A%2F%2Foss.sonatype.org)](https://oss.sonatype.org/content/repositories/snapshots/dev/zio/zio-kafka_2.13/)
 
+# ⚠️ CDK specific: How to work on this fork️ ⚠️
+
+⚠️ Before to start to work on this repo, be sure to read this paragraph, please. ⚠️
+
+We, at Conduktor, maintain a fork of the `zio/zio-kafka` repo so that we can release new versions as soon as we need them.
+
+For that purpose and to adapt this repo to our needs we created a new `master` branch, named `cdk-master`, which is configured to be our default branch.    
+This `cdk-master` is based on the `master` branch and MUST be rebased on it when the `master` branch is updated.
```

Comment:
So far it was named `conduktor-master` can we stick with that?

Reply:
`conduktor-master` is too long and the versions are named `xxx-cdk` so I prefer `cdk-master`

Reply:
> Also I wonder if merging master into conduktor-master will not be cleaner since the branch is meant to be shared.

I'm sorry I don't understand your suggestion. It seems to me that it's what I'm suggesting in my doc.

Reply:
>  This `cdk-master` is based on the `master` branch and MUST be **rebased** on it when the `master` branch is updated.

When you `rebase` a branch, you have to force push it after. This can be  an issue when another developer is pushing/pulling the same branch.

If you merge `master` into `cdk-master` instead of rebasing `cdk-master` on top of `master` then you don't have to force push `cdk-master`. Also I am afraid that when `cdk-master` will have a lot of commit in more than `zio/master` the rebase will be long to do.